product/dialogs/miq_dialogs/miq_provision_vmware_dialogs_template.yaml

```diff
@@ -435,7 +435,7 @@
           :auto_select_single: false
           :description: Name
           :required_method: :validate_placement
-          :required: true
+          :required: false
```

Comment:
I would suggest creating a separate PR for the dialog/workflow validation changes.  I suggested this change to @agrare for local testing but did not expect it to be part of the PR.